atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -243,20 +244,30 @@ def set_group_manifest_info(self, extra, worker_metadatas):
             repo = ImageName.parse(repositories[0]).to_str(registry=False, tag=False)
             # group_manifests added the registry, so this should be valid
             registries = self.workflow.push_conf.all_registries
+            digest_version = None
+            digest = None
+            # try to find the actual str digest value in the manifest digest
+            try:
+                for digest_version, digest in manifest_digest.default.items():
```

Comment:
Could we check it by `if hasattr(manifest_digest, 'default'):` instead catching `AttributeError`?

Reply:
I'm quite lost here. What are you trying to achieve here?

Reply:
Under certain unit test conditions, manifest_digest exists but is not a ManifestDigest.  In that case, this is a fallback to retrieve the digest and digest_version.

Reply:
Can we just make the tests use a ManifestDigest here?

Reply:
It was using the wrong Manifest digest version or something; I just removed it from the test.